tests/unit_tests/crypto.cpp

```diff
@@ -74,17 +74,10 @@ TEST(Crypto, Ostream)
   EXPECT_TRUE(is_formatted<crypto::key_image>());
 }
 
-TEST(Crypto, verify_32)
+TEST(Crypto, null_keys)
 {
-  // all bytes are treated the same, so we can brute force just one byte
-  unsigned char k0[32] = {0}, k1[32] = {0};
-  for (unsigned int i0 = 0; i0 < 256; ++i0)
-  {
-    k0[0] = i0;
-    for (unsigned int i1 = 0; i1 < 256; ++i1)
-    {
-      k1[0] = i1;
-      ASSERT_EQ(!crypto_verify_32(k0, k1), i0 == i1);
-    }
-  }
```

Comment:
Please look at the code changes carefully. This patch is only meant to add a new test `null_keys`, not to remove any existing ones.